plugins/inputs/mysql/mysql.go

```diff
@@ -24,6 +30,10 @@ var sampleConfig = `
   #
   # If no servers are specified, then localhost is used as the host.
   servers = ["tcp(127.0.0.1:3306)/"]
+  PerfEventsStatementsDigestTextLimit = 120
```

Comment:
Yes, the configurations file(my.cnf) must be changed as follows:
```
bind-address = 0.0.0.0
server-id = 2
master-host =  [private-IP-of-master]
master-user = [replication-username]
master-password = [replication-password]
master-connect-retry = 60
```
Must this requires another MySQL to be a master for this slave.